app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark mb-5">
+  <div class="container-fluid container">
+    <%= link_to 'Public Recipes', '#' , class: "navbar-brand"%>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
- [ ] I suggest that when you click on the `public recipes` link to take you to the list of public recipes